README.md

```diff
@@ -41,6 +41,13 @@ http_requests_total{namespace="b"}
 
 This is enforced for any case, whether a label matcher is specified in the original query or not.
 
+### Metadata endpoints
+
+Similar to query endpoint, for metadata endpoints `/api/v1/series`, `/api/v1/labels`, `/api/v1/label/<name>/values` the proxy inject the specified label in the `match` selector. 
+
+NOTE: At the moment of creation `/api/v1/labels`, `/api/v1/label/<name>/values` does not support `match[]` so they are disabled by default. Use `-enable-label-apis` flag to enable 
+those (see https://github.com/prometheus/prometheus/issues/6178 for tracking development).
```

Comment:
i think this is a sane default (I was wondering why we don't enable it by default) :+1:

Reply:
Because it gives false impression that tenancy is ensured, but literally implementation of label matcher does not exists yet (WIP PRs only)